modules/runtime/pages/monitoring.adoc

```diff
@@ -9,7 +9,7 @@ In this page, _Administrators_ and _Process Managers_ can have a complete view o
 [NOTE]
 ====
 
-*For Enterprise, Performance and Efficiency editions only.*
+*For Access, Enterprise, Performance and Efficiency editions only.*
```

Comment:
Not in Efficiency according to Support

Reply:
Ok, I let you modify it directly

Reply:
my mistake. this page is for process monitoring. so it is ok